ttg/ttg/parsec/ttg.h

```diff
@@ -765,7 +766,7 @@ namespace ttg_parsec {
         parsec_ttg_task_t<TT> *me = (parsec_ttg_task_t<TT> *)parsec_task;
         return me->template invoke_op<ttg::ExecutionSpace::CUDA>();
       } else {
-        throw std::runtime_error("PaRSEC CUDA hook invoked on a TT that does not support CUDA operations!");
+        return PARSEC_HOOK_RETURN_NEXT;
```

Comment:
It is now incorrect to return NEXT from a `hook`, it can be returned only during `evaluate`. This will cause parsec to invoke a fatal error. You shall return PARSEC_HOOK_RETURN_ERROR instead, but. that will also cause a fatal error, as it is too late in the game to change your mind (the data has been positioned on that device, and the owner device has been set).

Reply:
Will be fixed, thanks!